File: R/8kyu/8kyu-convert-a-boolean-to-a-string.r
# Convert a Boolean to a String
# Language: R
# Difficulty: 8 kyu

boolean_to_string <- function(b){
  toString(b)
}


# Tests
test_that('Basic tests', {
  expect_equal(boolean_to_string(TRUE), "TRUE")
  expect_equal(boolean_to_string(FALSE), "FALSE")
})

File: Python/8kyu/8kyu-convert-a-boolean-to-a-string.py
# Convert a Boolean to a String
# Language: Python
# Difficulty: 8 kyu

def boolean_to_string(b):
    return str(b)


# Tests
import codewars_test as test
from solution import boolean_to_string

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(boolean_to_string(True), "True")
        test.assert_equals(boolean_to_string(False), "False")